src/styles/global.scss

```diff
@@ -11,6 +11,7 @@
     --unit-100vh: 100vh;
     --unit-vertical-gap: 70px;
     --unit-gutter: 50px;
+    --unit-spacer: 17px;
```

Comment:
Don't think we need this as a global, this can be defined only in your component if you need it.